doc/source/reference/arrays.dtypes.rst

```diff
@@ -225,16 +231,18 @@ Array-protocol type strings (see :ref:`arrays.interface`)
    supported kinds are
 
    ================   ========================
-   ``'b'``            boolean
+   ``'?'``            boolean
+   ``'b'``            (signed) byte
+   ``'B'``            unsigned byte
```

Comment:
yes but it is imo not a good idea, the struct typecodes are the poor old C type where it is unknown what they actually mean.
The numpy ones i1, i2, i4, i8 are explicit and should be used instead.
so technically `b` and `B` should also not be in there but they do have some relevance in regards to the `S` type.